pkg/operator/ceph/object/user/controller.go

```diff
@@ -321,3 +413,50 @@ func updateCephObjectStoreUserStatus(name, namespace, status string, context *cl
 		return
 	}
 }
+
+// contains checks if an item exists in a given list.
+func contains(list []string, s string) bool {
```

Comment:
can we factor `contains()` and `remove()` to helper methods?

Reply:
sure!